ui/js/services/jobfilters.js

```diff
@@ -123,6 +123,24 @@ treeherder.factory('thJobFilters', [
             },
         };
 
+        const FILTER_GROUPS = {
+            failures: {
+                value: 'failures',
+                name: 'failures',
+                resultStatuses: thFailureResults.slice(),
+            },
+            nonfailures: {
+                value: 'nonfailures',
+                name: 'non-failures',
```

Comment:
Why does 'non-failures' name have a hyphen and the value does not? (since the other filter groups are consistent with name and value.)

Reply:
Heh, that's an excellent question.  The answer is that  ``name`` and ``value`` are vestigial artifacts of how we did things in Angular.  We don't need them anymore.  So I simplified that structure to just be an object with keys as each group (used by the colored fitler chicklets(dots) on the navbar) with values as arrays of ``resultStatus``.

Thanks for catching that! :)